ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/tree/node/services/TreeNodeFactoryImpl.java

```diff
@@ -24,8 +25,7 @@ public TreeNodeFactoryImpl(LdesFragmentRepository ldesFragmentRepository, Member
 
 	@Override
 	public TreeNode getTreeNode(LdesFragmentIdentifier treeNodeId, String hostName, String collectionName) {
-		String treeNodeIdString = treeNodeId.asString();
-		String extendedTreeNodeId = hostName + treeNodeIdString;
+		String extendedTreeNodeId = UrlCharachterCodesConverter.encode(hostName, treeNodeId);
```

Comment:
I find this a bit confusing, when you call a method called 'encode', I would not expect this to create an entire url with hostname + viewname + fragmentpairs. You have to look at the class and method to know what happens here. Maybe rename the class + method to be explicit what it does or only put in the encode method what actually gets encoded?